docs/06-features-and-components/01-steadypage-js-utility.md

```diff
@@ -8,7 +8,7 @@ Before             |  After
 
 ## Fastdom
 
-**Do not use fastdom** when inserting elements with steady page. The utility uses fastdom under the hood to read and write where required. It batches the insertion of elements, so that if multiple elements are queued in the same animation frame they will be inserted in the same fastdom.write and the position scrolled once to prevent excessive page jumping.
+**Do not use fastdom** when inserting elements with steady page. The utility uses fastdom under the hood to read and write where required. It batches the insertion of elements, so that if multiple elements are queued in the same animation frame they will be inserted in the same fastdom.mutate and the position scrolled once to prevent excessive page jumping.
```

Comment:
The steadypage utility was on for about half an hour and then we realised it was causing problems so I meant to fix it and that was like 4 years ago. 😆 (Not saying it's anything in this PR but probably something I can remove!)

Reply:
alpha4life